build-tools/automation/yaml-templates/stage-msbuild-emulator-tests.yaml

```
@@ -1,14 +1,17 @@
 # Runs MSBuild tests against a device running on macOS
 
 parameters:
+  stageName: msbuilddevice_tests
   job_name: 'mac_dotnetdevice_tests'
+  depensOn: mac_build
```

Small typo here is breaking the default `dependsOn` case, the same issue is also present below.

---

We will also need to consider the build caching aspect of the unified pipeline, these jobs will need to be skipped if the Android commit has already been built.

---

```suggestion
  dependsOn: mac_build
```
Got it, I could not see it